workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/WorkflowInterceptor.kt

```diff
@@ -259,6 +260,356 @@ public interface WorkflowInterceptor {
         handler: (CO) -> WorkflowAction<P, S, O>
       ) -> CR
     ): CR = proceed(child, childProps, key, handler)
+
+    public fun <CResult> onRemember(
```

Comment:
I think these need to be implemented for wrapped `RenderContext` in `ChainedWorkflowInterceptor` https://github.com/square/workflow-kotlin/blob/56e6ee061fa7bc508e7e4e0d5188b9945f61d24f/workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/internal/ChainedWorkflowInterceptor.kt#L130